### ISSUE TITLE
Better SGD hyper-parameters

### ISSUE THREAD

**Issue by ElanVB** [ElanVB/indaba-pracs-2019#13]
We should give SGD with decay better hyper-parameters so that this method is somewhat competitive in the “Putting it all into practice” section. Otherwise, it might look like this method can be ignored.

I have actually played around a bit and I think a learning rate of 0.5 and a decay 0.01 works quite well.

I was also thinking of adding the Momentum with decay example because it is just too easy to leave as an exercise and it shows that many of these techniques can be combined. Hyper-parameter values for that case that provide decent performance are a learning rate of 0.5, momentum of 0.9, and decay of 0.01.

**Comment by ElanVB** [comment]
The final loss plots now look something like this:

![image](https://user-images.githubusercontent.com/11617844/62826752-5e7f6400-bbc1-11e9-8c6c-b23fdf48ef5c.png)
